analyzers/rspec/vbnet/S107_vb.net.html

```diff
@@ -21,4 +21,22 @@ <h2>Compliant Solution</h2>
   ' ...
 End Function
 </pre>
+<h2>Exceptions</h2>
+<pre>
+Public Class BaseClass
+	Public Sub New(Param1 As Integer)
+
+	End Sub
+End Class
+
+Public Class ExtensionClass
+	Inherits BaseClass
+
+	Public Sub New(Param1 As Integer, Param2 As Integer, Param3 As Integer, Param4 As Integer, Param5 As Long)
+		MyBase.New(Param1) ' Compliant, the parameters intended for the base class constructor do not count in the sum of the parameter list.
+
```

Comment:
Shouldn't we remove the empty line?

Reply:
It's the same as C#, I'm fine with it for illustration purposes.